src/Series.cpp

```diff
@@ -1479,15 +1479,75 @@ namespace
     {
         if( config.json().contains( key ) )
         {
-            dest = json::asLowerCaseStringDynamic( config[ key ].json() );
+            auto maybeString =
+                json::asLowerCaseStringDynamic( config[ key ].json() );
+            if( maybeString.has_value() )
+            {
+                dest = std::move( maybeString.get() );
+            }
+            else
+            {
+                throw error::BackendConfigSchema(
```

Comment:
Does this one need a new internal include?

Reply:
`#include "openPMD/Error.hpp"` is already included